Main.java

```diff
@@ -9,7 +11,34 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //asks the user for the first x coordinate
+    System.out.println("Please enter the first x coordinate");
+    //initializes the first x coordinate
+    int xOne = input.nextInt();
+
+    //asks the user for the first y coordinate
+    System.out.println("Please enter the first y coordinate");
+    //initializes the first y coordinate
+    int yOne = input.nextInt();
+
+    //asks the user for the second x coordinate
+    System.out.println("Please enter the second x coordinate");
+    //initializes the second x coordinate
+    int xTwo = input.nextInt();
+
+    //asks the user for the second y coordinate
+    System.out.println("Please enter the second y coordinate");
+    //initializes the second y coordinate
+    int yTwo = input.nextInt();
+
+    //Calculates the slope of the line
+    final double formula = (yTwo - yOne) / (xTwo - xOne);
```

Comment:
Is it possible the have a point where the x or y points have decimal places? These should probably be doubles instead of ints. We know this because the output had a slope of 2.0 instead of 2. Otherwise, this is good looking code!